mlir-tensorrt/compiler/lib/Dialect/Plan/Transforms/CreateClosedRegions.cpp

```diff
@@ -351,70 +351,11 @@ static void remapAnalysisState(DataFlowSolver &solver, ValueRange originals,
   }
 }
```

Comment:
You have lots of changes to this pass, so there the test `test/Dialect/Plan/create-closed-regions.mlir` should also be updated/expanded.